src/App.tsx

```diff
@@ -1,44 +1,66 @@
-  fromResultData,
-  yearlyProjection
-} from './services/CalculationService'
 import { SolarPanelsPane } from './components/SolarPanelsPane'
-import { ROIBreakdown } from './components/ROIBreakdown'
+import { INITIAL_INPUT_DATA } from './constants'
+import { calculateResultData, fromResultData, yearlyProjection } from './services/CalculationService'
 
 export const App: React.FunctionComponent = () => {
   const { t, i18n } = useTranslation()
   const changeLanguage = (value: string) => { i18n.changeLanguage(value) }
   const [inputData, setInputData] = useState<InputData>(INITIAL_INPUT_DATA)
   const resultData = useMemo(() => calculateResultData(inputData), [inputData])
   const projection = useMemo(() => yearlyProjection(30, fromResultData(resultData)), [resultData])
+
+  const [current, setCurrent] = useState<number>(0)
+
   return (
     <div className="container">
-      <Card title={t('title')} extra={(
-        <Select size="small" onChange={changeLanguage} defaultValue={i18n.language.split('-')[0]}>
-          <Select.Option key="en" value="en">🇺🇸 EN</Select.Option>
-          <Select.Option key="id" value="id">🇮🇩 ID</Select.Option>
-        </Select>
-      )}>
-        <InputForm initialValue={INITIAL_INPUT_DATA} onChange={(data) => setInputData(data)} />
-      </Card>
-      <Card title={t('resultsTitle')}>
-        <SolarPanelsPane numberOfPanels={resultData.numberOfPanels} />
-        <ResultTable results={resultData} />
-      </Card>
-      <Card title={t('roiTitle')}>
-        <ROIChart yearly={projection} />
-        <Divider />
-        <ROIBreakdown yearly={projection} />
-      </Card>
+      <nav className="app-nav">
+        <div className="app-nav-logo"><Logo width={40} height={40} viewBox="0 0 32 32" /></div>
+        <Typography.Title ellipsis>{t('title')}</Typography.Title>
+        <div className="app-nav-extra">
+          <Select onChange={changeLanguage} defaultValue={i18n.language.split('-')[0]} bordered={false} style={{ color: '#FFFFFF' }} size="large">
+            <Select.Option key="en" value="en">🇺🇸 EN</Select.Option>
+            <Select.Option key="id" value="id">🇮🇩 ID</Select.Option>
+          </Select>
+        </div>
+      </nav>
+      <div className="card">
+        <div className="card-header">
+          <Steps size="small" current={current} onChange={setCurrent}>
+            <Steps.Step icon={<EditOutlined />} title={t('wizard.information.title')} />
+            <Steps.Step icon={<PieChartOutlined />} disabled={!inputData.pvOut} status={inputData.pvOut ? undefined : 'wait'} title={t('wizard.characteristics.title')} />
+            <Steps.Step icon={<DollarOutlined />} disabled={!inputData.pvOut} status={inputData.pvOut ? undefined : 'wait'} title={t('wizard.roi.title')} />
+          </Steps>
+        </div>
+        {current === 0 && <div className="card-body">
+          <InputForm initialValue={INITIAL_INPUT_DATA} onChange={(data) => setInputData(data)} />
+        </div>}
```

Comment:
I think this bit resets the state to the initial data when you take a step back in the wizard.

Reply:
I think hiding instead of removing the nodes fixed it